docs/DeveloperGuide.md

```diff
@@ -27,6 +27,13 @@ a Command Line Interface (CLI).
 
 ### 1.4 Typical User Workflow
 
+![Activity Diagram of DonnaFin](diagrams/ActivityDiagram.png)
+
+This Activity Diagram shows how one might navigate through the DonnaFin application in a typical usage.
```

Comment:
This Activity Diagram shows how one might typically navigate through the DonnaFin application.

Reply:
I believe activity diagram should be lower case. Not a class name or a proper noun, soz

Reply:
I believe activity diagram should be lower case. Not a class name or a proper noun, soz

Reply:
```suggestion
This activity diagram shows how one might typically navigate through the DonnaFin application.
```